From 409e0cdcd730a0fd064b148147e8b3d5a2fe6c71 Mon Sep 17 00:00:00 2001
From: Ani Sinha <anisinha@redhat.com>
Date: Fri, 17 Jan 2025 10:50:03 +0530
Subject: [PATCH] downstream: Revert "feat: Use NetworkManager renderer by
 default in RHEL family"

RH-Author: Ani Sinha <anisinha@redhat.com>
RH-MergeRequest: 120: downstream: Revert "feat: Use NetworkManager renderer by default in RHEL family"
RH-Jira: RHEL-74334
RH-Acked-by: xiachen <xiachen@redhat.com>
RH-Acked-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>
RH-Commit: [1/1] c466d30ee35576293011270591ec24773fc69251 (anisinha/cloud-init)

This reverts commit 97537494d95f733fce260beb5a2d465cccf9f6c7.

c9s has used sysconfig as its default network config renderer for historical
reasons. It should continue to do so. In RHEL-10, we can move to NM as its
default renderer. Therefore, revert the upstream change that makes NM as the
default for RHEL family.

X-downstream-only: true
Signed-off-by: Ani Sinha <anisinha@redhat.com>
---
 config/cloud.cfg.tmpl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/cloud.cfg.tmpl b/config/cloud.cfg.tmpl
index 82a7e0fe5..b4a6bc936 100644
--- a/config/cloud.cfg.tmpl
+++ b/config/cloud.cfg.tmpl
@@ -313,7 +313,7 @@ system_info:
     activators: ['netplan', 'eni', 'network-manager', 'networkd']
 {% elif is_rhel %}
   network:
-    renderers: ['eni', 'netplan', 'network-manager', 'sysconfig', 'networkd']
+    renderers: ['sysconfig', 'eni', 'netplan', 'network-manager', 'networkd']
 {% endif %}
 {% if variant == "photon" %}
   # If set to true, cloud-init will not use fallback network config.
-- 
2.39.3

